### PR TITLE
fix: variable cleanup

### PR DESCRIPTION
**Pull request by iamar7**
### Description

Stack Variable Cleanup

### Release required?

- [ ] No release
- [ ] Patch release (`x.x.X`)
- [x] Minor release (`x.X.x`)
- [ ] Major release (`X.x.x`)

### Run the pipeline

If the CI pipeline doesn't run when you create the PR, the PR requires a user with GitHub collaborators access to run the pipeline.

Run the CI pipeline when the PR is ready for review and you expect tests to pass. Add a comment to the PR with the following text:

```
/run pipeline
```

### Checklist for reviewers

- [ ] If relevant, a test for the change is included or updated with this PR.
- [ ] If relevant, documentation for the change is included or updated with this PR.

### For mergers

- Use a conventional commit message to set the release level. Follow the [guidelines](https://terraform-ibm-modules.github.io/documentation/#/merging.md).
- Include information that users need to know about the PR in the commit message. The commit message becomes part of the GitHub release notes.
- Use the **Squash and merge** option.

### CODE REVIEW

**Review by ocofaigh — APPROVED** [review]

File: ibm_catalog.json [hunk 1]
```diff
@@ -117,10 +117,10 @@
...
 								"description": "Creates and configures an IBM Security Compliance Center instance."
 							},
-              {
-                "title": "Creates an Security and Compliance Center Workload Protection instance",
-                "description": "Creates and configures an Security and Compliance Center Workload Protection instance."
-              },
+              				{
+                				"title": "Creates an Security and Compliance Center Workload Protection instance",
+                				"description": "Creates and configures an Security and Compliance Center Workload Protection instance."
+              				},
```
Comment (SirSpidey) [inline ibm_catalog.json:123]:
```suggestion
              				{
                				"title": "Creates a Security and Compliance Center Workload Protection instance",
                				"description": "Creates and configures a Security and Compliance Center Workload Protection instance."
              				},
```

older typo

**Review by SirSpidey — CHANGES_REQUESTED** [review]
typo from previous merges

File: ibm_catalog.json (hunk 1 above)
Comment (ocofaigh) [reply ibm_catalog.json:123]: @SirSpidey Ill fix it in https://github.com/terraform-ibm-modules/stack-ibm-core-security-services/pull/58 so the test dont have to run again in this PR

Comment (ocofaigh) [reply ibm_catalog.json:123]: @SirSpidey Fixed in https://github.com/terraform-ibm-modules/stack-ibm-core-security-services/pull/58 if you can re-review please?

**Review by SirSpidey — APPROVED** [review]
👍 Typo fixed in #58

**Comment by ocofaigh** [comment]
I'll generate the release with https://github.com/terraform-ibm-modules/stack-ibm-core-security-services/pull/58/

**Pull request merged**